src/util/package-branch-prefix.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// map from a packageName to the prefix used in release branch creation.
+export function packageBranchPrefix(packageName: string, releaseType?: string) {
```

Comment:
I kept the name more generically related to converting a package name into a branch prefix. I added the `releaseType` argument to support future releasers possibly needing to make a similar transformation.

Reply:
I like this approach 👍 I'd added some of this handling for Node.js, because I know the domain well, but I could certainly imagine us wanting similar behavior for other languages.